src/main/drivers/max7456.c

```diff
@@ -757,6 +760,37 @@ void max7456WriteNvm(uint8_t char_address, const uint8_t *font_data)
     max7456Lock = false;
 }
 
+void max7456ReadNvm(uint8_t char_address, uint8_t *font_data)
+{
+#ifdef MAX7456_DMA_CHANNEL_TX
+    while (dmaTransactionInProgress);
+#endif
+    while (max7456Lock);
+    max7456Lock = true;
+
+    __spiBusTransactionBegin(busdev);
+    // disable display
+    fontIsLoading = true;
+
+    max7456Send(MAX7456ADD_VM0, 0);                                         // disable display
+    max7456Send(MAX7456ADD_CMAH, char_address);                             // set start address high
+    max7456Send(MAX7456ADD_CMM, READ_NVR );                                 // copy from NVM to shadow ram
+    while ((max7456Send(MAX7456ADD_STAT, 0x00) & STAT_NVR_BUSY) != 0x00);   // wait until bit 5 in the status register returns to 0 (12ms)
+    for (int x = 0; x < NVM_RAM_SIZE; x++) {
+        max7456Send(MAX7456ADD_CMAL, x);                                    // select the offset address in the shadow RAM
+        *(font_data + x) = max7456Send(MAX7456ADD_CMDO_R, 0xFF);                // read the data from the shadow RAM
+#ifdef LED0_TOGGLE
+        LED0_TOGGLE;
+#else
+        LED1_TOGGLE;
+#endif
+    }
+
+    __spiBusTransactionEnd(busdev);
+
+    max7456Lock = false;
```

Comment:
github search doesn't return any place where fontIsLoading is set ... 

It will block OSD redraw after reading NVM, which is probably undesirable ...

Reply:
Looking at it I belive that fontIsLoading is initialized to false and it is only set to true when writing NVM - the configurator forcces a reboot after loading is complete. I don't think this is needed for reading NVM so I will remove.